services/scan/src/precinct_scanner_app.ts

```diff
@@ -618,6 +548,22 @@ export async function buildPrecinctScannerApp(
         });
         return;
       }
+
+      const electionDefinition = store.getElectionDefinition();
+      const precinctSelection = store.getPrecinctSelection();
+      const layouts = await store.loadLayouts();
+      assert(electionDefinition);
+      assert(precinctSelection);
+      assert(layouts);
+      interpreter.configure({
+        electionDefinition,
+        precinctSelection,
+        layouts,
+        testMode: store.getTestMode(),
+        markThresholdOverrides: store.getMarkThresholdOverrides(),
+        ballotImagesPath: workspace.ballotImagesPath,
+      });
```

Comment:
I was imagining going one level deeper here. If you look at interpreter, it's basically a very thin wrapper around the interpret functions that caches the config. I think we can delete that wrapper and simply make a function `interpret` that loads the config from the store and passes it to the interpret functions. Then the state machine can call that interpret function (it has access to the store already).

This works though, so I'm open to stopping here if you feel strongly.

Reply:
I don't feel strongly, but I want to close this, so will leave as-is.